src/context/FilterProjectsContext.jsx

```diff
@@ -34,7 +34,9 @@ export const FilterProjectsProvider = ({ children }) => {
   const [enableFollowScreen, setEnableFollowScreen] = useState(false)
   const [mapBbox, setMapBbox] = useState({})
 
-  const displayedSurveys = displayedProjects.flatMap((project) => project.records)
+  const filteredSurveys = displayedProjects
```

Comment:
renaming to differentiate from existing `displayedProjects` which appears to account for only some filters.